src/pages/derivx/_start-derivx.js

```diff
@@ -0,0 +1,255 @@
+        image_alt: 'real_step3',
+    },
+    {
+        description: <Localize translate_text="Start trading on the mobile app or through your web browser." />,
+        image_data: 'real_step4',
+        image_alt: 'real_step4',
+    },
+]
+
+const Section = styled(SectionContainer)`
+    display: flex;
+    flex-direction: column;
+    padding: 8rem 12rem;
+    align-items: center;
+    justify-content: center;
+
+    @media ${device.laptopM} {
+        padding: 8rem 6rem;
+    }
+    @media ${device.tabletS} {
+        padding: 40px 0;
+        height: auto;
+    }
+`
+const ImageWrapper = styled.div`
+    max-width: 79.2rem;
+    width: 100%;
+    height: 43.4rem;
+    position: relative;
+    margin: -3.2rem auto;
+
+    div {
+        width: 100%;
+    }
+    @media ${device.tabletS} {
+        max-width: 576px;
+        width: 100%;
+        margin: 0 0 24px;
+        height: unset;
+
+        div {
+            max-width: 576px;
+            width: 100%;
+        }
+    }
+`
+const TabItem = styled.div`
+    padding: 2.4rem 4rem;
+    width: fit-content;
+    height: 8.4rem;
+    border-radius: 4px;
+    border: solid 1px rgba(51, 51, 51, 0.1);
+    cursor: pointer;
+    ${(props) =>
+        props.active
+            ? css`
+                  box-shadow: 0 16px 20px 0 rgba(0, 0, 0, 0.05), 0 0 20px 0 rgba(0, 0, 0, 0.05);
+                  border: unset;
+                  ${Text} {
+                      font-weight: bold;
+                  }
+              `
+            : css`
+                  box-shadow: unset;
+                  ${Text} {
+                      font-weight: unset;
+                  }
+              `}
+
+    @media ${device.tabletS} {
+        padding: 17px 20px;
+    }
+    @media ${device.mobileL} {
+        max-width: 164px;
+        width: 100%;
+        padding: ${(props) => props.mobile_padding};
+    }
+`
+const StyledHeader = styled(Header)`
+    @media ${device.mobileL} {
+        font-size: 32px;
+        margin-bottom: 24px;
+        padding: 0 16px;
+    }
+`
+const StyledText = styled(Text)`
+    @media ${device.mobileL} {
+        font-size: 16px;
+    }
+`
+
+const StartDerivX = () => {
+    const [is_mobile, setMobile] = useState(false)
+    const handleResizeWindow = useCallback(() => {
+        setMobile(isBrowser() ? window.screen.width <= size.tablet : false)
+    }, [setMobile])
+
+    useEffect(() => {
+        handleResizeWindow()
+        window.addEventListener('resize', handleResizeWindow)
+
+        return () => {
+            window.removeEventListener('resize', handleResizeWindow)
+        }
+    }, [handleResizeWindow])
+
+    const data = useStaticQuery(query)
+    const [tab, setTab] = useState('demo')
+
+    const onTabClick = (tab) => {
+        setTab(tab)
+    }
+
+    return (
+        <Section>
+            <StyledHeader align="center" mb="4rem" as="h2" type="page-title">
+                {localize('How to get started with a Deriv X account')}
```

Comment:
Is there a question mark here?

Reply:
According to design there's no question mark here